### ISSUE TITLE
Dark footer link text needs better contrast

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-news-2021#158]
Ex, on the Releases page. The regular text switches to white, but the link text is the same blue & does not have enough contrast against the dark background.

<img width="674" alt="Screen Shot 2022-01-06 at 5 31 34 PM" src="https://user-images.githubusercontent.com/541093/148461554-04bf1ecc-e381-49ad-90cc-98fdf4f51d37.png">
